plugin.yml

```
@@ -3,7 +3,7 @@ main: me.EtienneDx.RealEstate.RealEstate
 version: ${project.version}
 authors: [EtienneDx]
 depend: [GriefPrevention, Vault]
-api-version: "1.15"
+api-version: '1.13'
```

Considering that GriefPrevention has moved on from 1.13 towards 1.15, the latest versions of RealEstate need to match it, so we should keep it as 1.15
```suggestion
api-version: '1.15'
```